tests/RAGchain/utils/linker/test_base_linker.py

```diff
@@ -1,8 +1,10 @@
 import pytest
+from uuid import uuid4
 
 from RAGchain.utils.linker import DynamoLinker, RedisLinker, SingletonCreationError
 
-TEST_IDS = ['test_id_1']
+TEST_UUID_IDS = [uuid4()]
+TEST_STR_IDS = [str(TEST_UUID_IDS[0])]
```

Comment:
Recommend test non-uuid string.

Reply:
If I input by uuid and search by str(non-uuid), or vice versa, I should get a 'NoIdWarning'. Isn't this a duplicate test of test_no_id_warning?